firmware/integration/rusefi_config.txt

```diff
@@ -1954,7 +1954,7 @@ end_struct
 #define ENGINE_SNIFFER_UNIT_US 10
 
 ! These commands are used exclusively by the rusEfi console
-#define TS_TEST_COMMAND 't' // 0x74
```

Comment:
this change removes a valuable in my opinion comment. please do not remove the valuable in my opinion comment

Reply:
we never refer to this by anything other than char value `'t'`, but ok